src/main/java/com/kakao/cafe/controller/UserController.java

```diff
@@ -0,0 +1,55 @@
+package com.kakao.cafe.controller;
+
+import com.kakao.cafe.domain.User;
+import com.kakao.cafe.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.List;
+
+@Controller
+public class UserController {
+
+    private final UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
```

Comment:
문서를 살펴보니 `If a class only declares a single constructor to begin with, it will always be used, even if not annotated.` 라고 적혀있어서 생성자가 한개인 클래스라면 어노테이션이 없어도 동작하도록 되어있다고 이해했습니다. 혹시 틀렸다면 지적해주시고 이보다 더 구체적인 이유가 있다면 더 찾아보고 공부해보겠습니다!